client/src/components/ProductDetails/ImageThumbnail.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import sampleImage from './sampleData/cat.png'
```

Comment:
I see that ImageThumbnail is using its props as the source image instead of using sampleImage.
Did you mean to leave this line of code?